### PR TITLE
Batch dml mainline

### PR DESCRIPTION
**Pull request by guangyus**
Fixes #<issue_number_goes_here> (it's a good idea to open an issue first for context and/or discussion)

### CODE REVIEW

File: google-cloud-clients/google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerBatchUpdateException.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2017 Google LLC
```
Comment (kolea2) [inline google-cloud-clients/google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerBatchUpdateException.java]: Can you please update the year here?

File: google-cloud-clients/google-cloud-spanner/src/test/java/com/google/cloud/spanner/it/ITBatchDmlTest.java [hunk 2]
```diff
@@ -0,0 +1,218 @@
+/*
+ * Copyright 2017 Google LLC
```
Comment (kolea2) [inline google-cloud-clients/google-cloud-spanner/src/test/java/com/google/cloud/spanner/it/ITBatchDmlTest.java]: same here please

**Comment by kolea2** [comment]
@guangyus, looks like the code formatting build is failing. Please see here under 'Coding Style' for how to format your code: https://github.com/googleapis/google-cloud-java/blob/master/CONTRIBUTING.md. Thanks!

File: google-cloud-clients/google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerBatchUpdateException.java (hunk 1 above)
Comment (guangyus) [reply google-cloud-clients/google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerBatchUpdateException.java]: Done. Thanks!

File: google-cloud-clients/google-cloud-spanner/src/test/java/com/google/cloud/spanner/it/ITBatchDmlTest.java (hunk 2 above)
Comment (guangyus) [reply google-cloud-clients/google-cloud-spanner/src/test/java/com/google/cloud/spanner/it/ITBatchDmlTest.java]: Done.

**Comment by guangyus** [comment]
> @guangyus, looks like the code formatting build is failing. Please see here under 'Coding Style' for how to format your code: https://github.com/googleapis/google-cloud-java/blob/master/CONTRIBUTING.md. Thanks!

Fixed by running mvn com.coveo:fmt-maven-plugin:format.

**Pull request merged**